packages/app-desktop/gui/hooks/useToast.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { useContext, useCallback } from 'react';
+import { PopupNotificationContext } from '../PopupNotification/PopupNotificationProvider';
+import { ToastOptions, NotificationType } from '../PopupNotification/types';
+import type { PopupHandle } from '../PopupNotification/types';
+
+export default function useToast() {
+	const popupManager = useContext(PopupNotificationContext);
+
+	return useCallback(
+		(message: React.ReactNode | string, options: ToastOptions = {}) => {
+			const {
+				type = NotificationType.Info,
+				delay = 4000,
+				persistent = false,
+			} = options;
+
+			const handle: PopupHandle = popupManager.createPopup(() => message, { type });
```

Comment:
Please don't define types that can inferred - https://joplinapp.org/help/dev/coding_style/#dont-set-the-type-when-it-can-be-inferred